### ISSUE TITLE
SVG Pixelated when scaling

### ISSUE THREAD

**Issue by ddreharov** [Automattic/node-canvas#1774]
## Issue or Feature
I'm sorry if there is a duplicated issue for this.

The issue is reproducible only when resizing SVG image. It becomes pixelated/blurry/bad quality etc.  In the browser this issue could not be reproduced. 

## Steps to Reproduce

```js
const fs = require('fs');
const { createCanvas, loadImage } = require('canvas');

(async function() {
    const canvas = createCanvas(1920, 1080);
    const ctx = canvas.getContext('2d');
    const scale = 8;

    ctx.fillStyle = "white";
    ctx.fillRect(0, 0, canvas.width, canvas.height);

    const image = await loadImage('./21.svg');

    ctx.drawImage(image, 0, 0, image.naturalWidth, image.naturalHeight, 0, 0, image.naturalWidth * scale, image.naturalHeight * 8);
    const stream = canvas.createPNGStream();
    stream.pipe(fs.createWriteStream('output.png'));
})()

```

## Your Environment
* Version of node-canvas: canvas@2.7.0
* Environment: v12.19.0 on Linux Mint, same results for alpine node v14

**Comment by zbjornson** [comment]
Thanks for the clear repro instructions. I think this is a duplicate of #1555 (which I also marked as a duplicate, but I don't remember of what issue). Can you read through that and confirm please?

**Comment by ddreharov** [comment]
Yes, I think we pretty much describe the same issue. I saw a no updates of them since last year. Do you have any?

btw I've noticed this problem when I was experimenting with fabric.js which depends on this library. Probably many other libraries depending on this one suffer from it too :/
